Swipe navigation

testower opened the pull request:
Hopefully solves #15 as a first draft

* Rewrote arrow navigation to use hooks
* Similarly implemented a new hook for swipe navigation
  - Touch event listeners attached to window, maybe consider attaching to more specific element?
  - Mouse swiping not supported in this implementation, but could be added if desired
* Removed two image files that I think were re-added by mistake

testower committed:
Proposal to rewrite PageFlipper to use hooks, to make swipe transitions easier to implement alongside arrow navigation

testower committed:
Draft hook for swipe navigation

testower committed:
Remove files that were probably added by mistake

testower committed:
Make sure it is not possible to attempt navigation beyond bounds

raae commented:
Love it! Tested on mobile phone and it works nicely. 

No need for mouse swipe at this moment. Other things are more important to finish before the deadline.

raae commented on theme/src/components/PageFlipper/useArrowNavigation.js:
```diff
@@ -0,0 +1,21 @@
+    return () => {
+      window.removeEventListener("keydown", handleKeyStroke)
+    }
+  })
```
Should we add an empty array here? 
So that this happens only happens onMount, not on every render?

testower replied:
Yeah, that makes sense. I tried adding it in useSwipeNavigtion, but there everything stopped working, which I don't really understand.

raae reviewed (commented):
Love it!

testower committed:
Add empty array to prevent effect from being executed on every render.

testower committed:
Remove preventDefault called, due to passive event on window

Could also be solved by passing passive false as option and/or not attaching event to window.

The motivation for preventDefault here is to prevent scrolling (?)
